V2_MIGRATION_GUIDE.md

```diff
@@ -50,6 +52,8 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
 - The `com.auth0.android.request.ParameterizableRequest` interface has been removed. The ability to add request headers and parameters has been moved to the `com.auth0.android.request.Request` interface.
 - The `com.auth0.android.request.AuthRequest` interface has been removed. The `com.auth0.android.request.AuthenticationRequest` interface can be used instead.
 - The `com.auth0.android.provider.WebAuthActivity` class has been removed. External browser applications will always be used for authentication.
+- The `com.auth0.android.result.Delegation` class has been removed. There's no replacement for it.
+- The `com.auth0.android.authentication.request.DelegationRequest` class has been removed. There's no replacement for it.
```

Comment:
Similar to above, even though there is no replacement would be good to provide context for why it's been removed (e.g., uses legacy endpoints or similar).